src/containers/Application/NavigationBox.tsx

```diff
@@ -54,6 +64,13 @@ const NavigationBox: React.FC<NavigationBoxProps> = (props) => {
     } else sendToPage(currentSection.code, nextPage)
   }
 
+  //
```

Comment:
Missing a comment here? I think that's a good idea to describe the use here for this `useEffect`